vizro-core/docs/pages/user-guides/card-button.md

```diff
@@ -402,63 +404,19 @@ Use the following pre-defined URL hashes in your image path to apply Vizro's def
     === "Result"
         [![CardImageFloating]][cardimagefloating]
 
-#### Card with icon
-
-- default icon styling (`icon-top`): `![](my_image.png#icon-top)`
-
-!!! example "Card with icon"
-    === "app.py"
-        ```{.python pycafe-link}
-        import vizro.models as vm
```

Comment:
@stichbury - I've removed the entire section, because the content of this is already covered by the subsections before:
- Place an image on a card
- Style a card image
- Make an icon responsive to theme switch

Reply:
This is fine. From what I can tell, the py.cafe is removed so we don't need to check that it's still OK with its changed code snippets, as the code snippets are fully removed.